core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -160,6 +160,7 @@ def _fit(self, X_train, y_train, k_fold=5, k_fold_start=0, k_fold_end=None, n_re
             self._add_child_times_to_bag(model=model_base)
             return
 
+        sample_weights = kwargs.get('sample_weights', None)
```

Comment:
`sample_weight` , `sample_weight_val` instead of `sample_weights`, `sample_weights_val`: we call it `sample_weight` elsewhere, and it is `sample_weight` in sklearn.

This comment applies to all usages of `sample_weights`